docs/howto/operator/gcp/gcs.rst

```diff
@@ -38,10 +38,10 @@ Use the
 :class:`~airflow.providers.google.cloud.operators.gcs_to_bigquery.GCSToBigQueryOperator`
```

Comment:
This guide should be in a separate file, but this is another problem. Each module (* .py) with operators should have a separate guide and separate unit test file and separate system test file.